client/src/Components/InterviewQuestionDetails.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from "react";
```

Comment:
We might not need useContext here, it's good to try and minimize imports as a best practice/bring them in as you need them.